Reactive statements throwing TypeError from svelte plugin

I just ran into an issue with Svelte with the reactive statements. I created a sample app using the `sveltejs/template` starter repo. I added a single reactive statement to the `App.svelte` component, which you can [see here](https://github.com/reinink/my-svelte-project/blob/aa0e5d425e405b40ac351b0094ca0dfb7939b610/src/App.svelte#L4)

```html
<script>
	export let name;

	$: document.title = name;
</script>

<style>
	h1 {
		color: purple;
	}
</style>

<h1>Hello {name}!</h1>
```

Here is the error I'm getting:

```
1!] (svelte plugin) TypeError: extractors[param.type] is not a function
npm ERR! code ELIFECYCLE
npm ERR! errno 1ctors[param.type] is not a function
npm ERR! svelte-app@1.0.0 autobuild: `rollup -c -w`y-svelte-project/node_modules/svelte/compiler.js:17623:28)
npm ERR! Exit status 1/Users/jonathan/Sites/my-svelte-project/node_modules/svelte/compiler.js:17619:13)
npm ERR! ript.content.body.forEach.node (/Users/jonathan/Sites/my-svelte-project/node_modules/svelte/compiler.js:22946:14)
npm ERR! Failed at the svelte-app@1.0.0 autobuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.es/svelte/compiler.js:22938:30)
    at new Component (/Users/jonathan/Sites/my-svelte-project/node_modules/svelte/compiler.js:22594:15)
npm ERR! A complete log of this run can be found in:ect/node_modules/svelte/compiler.js:23614:24)
npm ERR!     /Users/jonathan/.npm/_logs/2019-04-23T13_46_58_987Z-debug.logt/node_modules/rollup-plugin-svelte/index.js:236:22)
ERROR: "autobuild" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svelte-app@1.0.0 dev: `run-p start:dev autobuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svelte-app@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Full repo [here](https://github.com/reinink/my-svelte-project).

--- a/src/compile/utils/scope.ts
+++ b/src/compile/utils/scope.ts
@@ -106,7 +106,7 @@ export function extract_names(param: Node) {
 
 export function extract_identifiers(param: Node) {
 	const nodes: Node[] = [];
-	extractors[param.type](nodes, param);
+	extractors[param.type] && extractors[param.type](nodes, param);
 	return nodes;
 }
 

--- a/test/runtime/samples/reactive-value-mutate/_config.js
+++ b/test/runtime/samples/reactive-value-mutate/_config.js
@@ -1,0 +1,3 @@
+export default {
+	html: `{"bar":42}`
+};

--- a/test/runtime/samples/reactive-value-mutate/main.svelte
+++ b/test/runtime/samples/reactive-value-mutate/main.svelte
@@ -1,0 +1,7 @@
+<script>
+	let foo = {};
+	let bar = 42;
+	$: foo.bar = bar;
+</script>
+
+{JSON.stringify(foo)}

